repos/spack_repo/builtin/packages/shred/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.makefile import MakefilePackage
+
+from spack.package import *
+
+
+class Shred(MakefilePackage):
+    """Stochastic and Hybrid Representation Electronic structure by Density
+    functional theory (SHRED). SHRED is a plane-wave DFT code similar to
+    ABINIT, VASP and Quantum Espresso."""
+
+    url = "https://github.com/alwhite-LANL/SHRED/"
+    git = "git@github.com:alwhite-LANL/SHRED.git"
+
+    maintainers("finkeljos", "alwhite")
+
+    license("GPL-3.0-or-later")
+
+    version("develop", branch="new_develop")
+
+    depends_on("fortran", type="build")
+    depends_on("c", type="build")
+
+    # Necessary package dependencies
+    depends_on("mpi")
+    depends_on("libxc")
+    depends_on("fftw+mpi+openmp")
+    depends_on("scalapack")
+    depends_on("lapack")
+    depends_on("blas")
```

Comment:
```suggestion
    depends_on("gmake", type="build")
    
    depends_on("blas")
    depends_on("fftw+mpi+openmp")
    depends_on("lapack")
    depends_on("libxc")
    depends_on("mpi")
    depends_on("scalapack")
```

Added `gmake` to the dependencies listed here. Alphabetized dependencies.